### DISCUSSION TITLE
timezone issue

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#3547] [Help]
Hello,

I am using MySQL DB and have many file with data type `DATE`, the data stored as `YYYY-MM-DD` without timezone info. Then problem is when I bind to the Date Picker, it always add 8hrs(my server local timezone is UTC +8) which caused incorrect date. I know it can be convert use something like `{{date field "YYYY-MM-DD" "UTC"}}` if I use repeater, but in many case I use the update form and there no option to convert the date format. Is there any way to globally enable or disable the date convert? Or globally change the timezone?

![image](https://user-images.githubusercontent.com/29649651/143475161-1d45269c-96e4-4c52-b17c-e39e5fb44270.png)

**Comment by mjashanks** [comment]
We do have an open issue with this (I have just reopened). https://github.com/Budibase/budibase/issues/1504

It's all quite confusing :) ...timezones!

**Reply by samg666** [reply]
This issue happens when using `Save Row` as Budibase treat date and date time fields all as the same `date time`, and I didn't find a way to globally fix it. The current workaround is don't use `Save Row`, use custom SQL to update the row, then I can manually convert the date by `{{date field "YYYY-MM-DD"}}`.
